_config.yml

```
@@ -1,16 +1,21 @@
 # Build settings
 permalink: /blog/:year/:title/
+permalink: /projects/:year/:title/
```

yamllint

[error] duplication of key "permalink" in mapping

(key-duplicates)